src/components/field/Field.vue

```diff
@@ -1,17 +1,33 @@
 <template>
     <div class="field" :class="[fieldType, newPosition, {
         'is-expanded': expanded,
-        'is-grouped-multiline': groupMultiline
+        'is-grouped-multiline': groupMultiline,
+        'is-horizontal': horizontal
     }]">
-        <label class="label" :for="labelFor" v-if="label">{{ label }}</label>
-        <slot></slot>
+        <div class="field-label is-normal" v-if="horizontal">
```

Comment:
The "is-normal" class add the required `padding-top` for aligning the label to the control text